tracer/src/Datadog.Trace/Iast/HashBasedDeduplication.cs

```diff
@@ -24,15 +24,15 @@ internal class HashBasedDeduplication
     /// </summary>
     internal HashBasedDeduplication(DateTime? currentTime = null)
     {
-        _cacheClearedTime = currentTime ?? DateTime.Now;
+        _cacheClearedTime = currentTime ?? DateTime.UtcNow;
```

Comment:
Can you add the reason for this change to the implementation details? It doesn't seem to relate to the rest of the PR.

Reply:
Yes, it is a minor fix made in the hashBasedDeduplication. I could maybe add it to a separate PR. It makes sure that we avoid problems related to daylight saving time.

Reply:
I'm fine with it being in the same PR, but we should just note that in the summary, like with the stringbuilder bug.

Reply:
Done!